src/view/screens/Profile.tsx

```diff
@@ -167,6 +170,9 @@ function ProfileScreenLoaded({
   const [currentPage, setCurrentPage] = React.useState(0)
   const {_} = useLingui()
   const setDrawerSwipeDisabled = useSetDrawerSwipeDisabled()
+  const gate = useGate()
+  const starterPacksEnabled =
+    IS_DEV || IS_TESTFLIGHT || (!isWeb && gate('starter_packs_enabled'))
```

Comment:
We will have to deploy `/start` to production for review. So gating based on `isWeb` as well as statsig here and we can remove that in a separate commit before releasing.

Native will remain ungated during review. Probably won't end up needing the statsig gate but just in case.

Reply:
So in prod this is always false on web, right, and it's false on native too unless the gate is enabled. So are we going to ramp to 100% for review?

Reply:
yup discussed on slack. will have this at 100% for review.